tests/30-main.spec.js

```diff
@@ -254,4 +255,450 @@ describe('main', () => {
       credential, documentLoader, verifyRevocationListCredential: false});
     result.verified.should.equal(false);
   });
+
+  it('should fail to verify status on missing credential param', async () => {
```

Comment:
This makes it more clear that the param is literally "credential" vs. some other kind of "credential parameter".

```suggestion
  it('should fail to verify status on missing "credential" param', async () => {
```